guides/common/modules/proc_importing-suse-products.adoc

```diff
@@ -1,15 +1,20 @@
 [id="Importing_SUSE_Products_{context}"]
 = Importing SUSE Products
 
-.Prerequisites
-* xref:Installing_the_SCC_Manager_Plugin_{context}[SCC Manager plug-in installed] on your {Project}
-* Access to the SUSE product `SUSE Linux Enterprise Server 15 SP3 x86_64`
+You can use this procedure to import producs from SUSE into {Project}.
+
+.Prerequisite
+* You have added your SCC account to {Project}.
+For more information, see xref:Adding_an_SCC_Account_to_Server_{context}[].
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Content > SUSE Subscriptions*.
 . Click *Select products* on your previously synchronized SCC account.
+. Use the *Select Product*, *Select Version*, and *Select Architecture* drop down menus to filter all available SUSE products.
+. Click *Search* to apply the filter.
 . Select all SUSE products you want to synchronize to {Project}.
-This guide assumes you have access to and select the `SUSE Linux Enterprise Server 15 SP3 x86_64` product.
-. Click *Submit* to create the selected SUSE products in {Project}.
-
-Continue with xref:Synchronizing_SUSE_Content_{context}[synchronizing SUSE products].
+By default, _debug_ and _source_ repositories are excluded.
+. Click *Add product(s)* to create the selected SUSE products in {Project}.
```

Comment:
I think it is worth mentioning that the page is not automatically reloaded once the products are imported in Katello. The page needs to be reloaded manually.

Reply:
I believe this is acceptable because the procedures is finished/ could continue if the user wants to follow the task. To add another product, they'd have to "restart" the whole procedure.